tests/data_io/test_connect.py

```diff
@@ -91,6 +91,7 @@ def test_get_export_config_default():
         "mergedArrayValuesDelimiter": "|",
         "limitRecords": 0,
         "columnsExcludeRegex": "",
+        "renameFields": None,
```

Comment:
Can you change the default to an empty Dict ("{}") to match the connect repo examples: https://github.com/Datatamer/core-connect#client-credentials-example-1

Reply:
Done